sdk/parents/clientcore-parent/pom.xml

```diff
@@ -958,7 +958,7 @@
               </configurationFile>
             </analysisConfigurationFiles>
             <versionFormat>^\d+\.\d+\.\d+$</versionFormat>
-            <checkDependencies>true</checkDependencies>
+            <checkDependencies>false</checkDependencies>
```

Comment:
We don't have any external dependencies in `core`. So, why would this cause an issue for clientcore? We still want this check if the transitive dependencies are owned by Azure SDK.

Reply:
We've disabled this wholesale in `azure-client-sdk-parent` a while back. This is just replicating that in ClientCore.

The root issue is that `io.clientcore:http-stress` uses `com.azure:perf-test-core` which pulls through transitive dependencies for Netty and a few other libraries. These then get flagged as illegally exposed 3rd party APIs, this doesn't happen in `com.azure` libraries as Netty is allowed.

Reply:
Yeah, fine with disabling. Also, we should disable running revapi on packages that are not released like `perf-test-core` and `http-stress`.